### PR TITLE
[docs] Adding `hasLoaded` property

### PR DESCRIPTION
**Pull request by delapuente**
`hasLoaded` property is missing from `entity` documentation.

### CODE REVIEW

File: docs/core/entity.md [hunk 1]
```diff
@@ -91,6 +91,22 @@ Or if a component exposes an API, we can call its methods:
...
+ ```js
+ function doSomething() {
+   if (!entity.hasLoaded) {
+     entity.addEventListener('loaded', doSomething);
+     return;
+   }
+   // do something with entity
+ }
+ var entity = document.querySelector('a-entity');
+ doSomething();
```
Comment (donmccurdy) [inline docs/core/entity.md]: To encourage people to write their JS using components, could I suggest reorganizing this as:

```js
AFRAME.registerComponent('my-behavior', {
  init: function () {
    var el = this.el;
    if (el.hasLoaded) {
      this.doSomething();
    } else {
      el.addEventListener('loaded', this.doSomething.bind(this));
    }
  },
  doSomething: function () {
    // ...
  }
});
```

Comment (ngokevin) [reply docs/core/entity.md]: was literally just changing to something like that

Comment (ngokevin) [reply docs/core/entity.md]: though even then i'm not sure if that pattern is appropriate. better to explicitly wait on something or add a dependency, i don't know how common it is to run something on `loaded`

Comment (donmccurdy) [reply docs/core/entity.md]: ^For a scene component I use this. For normal components more often using a dependency, `model-loaded`, or `object3dset`, yeah.

**Pull request merged**